### ISSUE TITLE
Scaling

### ISSUE THREAD

**Issue by ralcr** [fikovnik/ShiftIt#226]
Hi,
Would be possible to scale the entire window like expose on the mac is doing?

**Comment by fikovnik** [comment]
The maximize is not the same?

**Comment by ralcr** [comment]
No, expose feature is scaling the windows not changing their size.

**Comment by fikovnik** [comment]
Can you be a bit more specific, maybe I'm dumb, but I'm bot sure what you are after

**Comment by ralcr** [comment]
Maybe you are new to Mac and you don't know what expose means? It's when the apps are scaled down so you can see them all at once, the feature is called mission control now. https://en.m.wikipedia.org/wiki/Mission_Control_(macOS) thanks

**Comment by VadimKulagin** [comment]
Can you describe the use case? What is it for? And it's better than resizing the window? I use Expose & Mission Control every day, but do not understand why you need it for one window.

**Comment by ralcr** [comment]
Sure, i want to make something like a picture-in-picture, like the videos in sierra, for apps. I think scaling looks and feels more natural than messing with the content of the window. I do not want to use them when they are scaled down but to have quicker access to them.

**Comment by fikovnik** [comment]
Well, pull request is very welcome :-)
